test/functional/tools/discover_sort_by.xml

```diff
@@ -0,0 +1,26 @@
+<tool id="collection_split_on_column" name="collection_split_on_column" version="0.1.0">
```

Comment:
You have to add the tool to test/functional/tools/samples_tool_conf.xml to run the tests

Reply:
Hah. Just forgot to commit it.